tests/integration/test_lists/test-db/l0_a10.yml

```diff
@@ -22,14 +27,14 @@ l0_a10:
   - test_cpp.py::test_model[recurrentgemma-86]
   - test_cpp.py::test_model[eagle-86]
   # ------------- TRT tests ---------------
-  - unittest/trt/attention/test_gpt_attention.py -k "partition0"
```

Comment:
There shouldn't be changes to this file other than addition of `stress_test/stress_test.py` tests.

Reply:
Let me update